### ISSUE TITLE
Potential bug in syncing NCMC and MD simulation

### ISSUE THREAD

**Issue by nathanmlim** [MobleyLab/blues#149]
@tannerbobak reports (copied below) a potential issue in between iterations where the MD simulation appears to spike in energy:

> The last 10 steps of MD output before the crash were as follows, where steps recorded starting on step 880000 immediately followed step 10000 above, presumably:

```
#"Step","Time (ps)","Potential Energy (kilocalorie/mole)","Kinetic Energy (kilocalorie/mole)","Total Energy (kilocalorie/mole)","Temperature (K)","Box Volume (angstrom**3)"
# ...
879996,3519.983999944411,-91464.86295702822,17970.967140484438,-73493.89581654378,300.73763270124044,297116.3310449951
879997,3519.987999944411,-91363.01005604128,17887.521051892058,-73475.48900414922,299.34119260175817,297116.3310449951
879998,3519.991999944411,-91387.74779088747,17893.830510174816,-73493.91728071265,299.446779103183,297116.3310449951
879999,3519.995999944411,-91370.25827431864,17877.93989200145,-73492.3183823172,299.1808553577185,297116.3310449951
880000,3519.999999944411,-91360.78369159023,17862.111587752162,-73498.67210383806,298.91597441322654,297116.3310449951
880001,3520.0039999444107,-92091.4895172249,25208.153927306386,-66883.3355899185,421.8493349636296,297116.3310449951
880002,3520.0079999444106,-92057.50197513541,25179.080638134208,-66878.42133700121,421.3628039888563,297116.3310449951
880003,3520.0119999444105,-92037.01535083924,25044.845840687816,-66992.16951015142,419.1164332234668,297116.3310449951
880004,3520.0159999444104,-91194.65670458875,24275.088384701427,-66919.56831988732,406.2348207171499,297116.3310449951
880005,3520.0199999444103,-90594.1070057065,25081.83562804012,-65512.27137766637,419.7354439308758,297116.3310449951
```
> MD is paused at 880000, and NCMC occurs (specifically, the 10K steps of NCMC posted above), then MD picks back up at 880001. Summarizing, it looks like on the 88th iteration the kinetic energy suddenly jumped by 7000 kcal/mol and the temperature by 100K. 

This suggests a potential issue in the syncing function located [here]( https://github.com/MobleyLab/blues/blob/878cf64ad0fa5aa3cf646af3ac4bb655ba476449/blues/simulation.py#L828)

**Comment by nathanmlim** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Potential bug in syncing NCMC and MD simulation !!! [iss149]](https://trello.com/c/GuSrJA2N/89-potential-bug-in-syncing-ncmc-and-md-simulation-iss149)

**Comment by nathanmlim** [comment]
Closing. Solved in #150
